### ISSUE TITLE
Randomness in Frame Selection - CREMAD

### ISSUE THREAD

**Issue by kkontras** [GeWu-Lab/OGM-GE_CVPR2022#41]
Hi,

Thank you for the support in this repo. I noticed that on CramedDataset in __get_item__ you pick each time randomly the frame that you are using through np.random.choice. I find that strange since this pytorch dataset is used both for training and testing. Eventually that would mean that in your training set you use  all images but on each datapoint a different one (I guess that would be a form of augmentation) and during test time there is some ambiguity to the results and possibly they could change with different seed. Am I missing something?

**Comment by wang-MIG-CFM-UESTC** [comment]
Hello, I have a similar problem with the frames selection. Noticed **np.random.choice** is used in the code, but **select_index** does not to be used. Does this setting need to be changed manually during training and testing phases?
